client/state/products-list/selectors.js

```diff
@@ -1,3 +1,7 @@
+import { property } from 'lodash';
+
+export const getProductsList = property( 'productsList.items' );
```

Comment:
uh…what do you think we should test?

I think the test would end up being circular, as in we'll either mock out this very structure and test that it matches (which doesn't actually test anything but does add maintenance overhead) or try and instantiate the entire Redux tree inside the test (which is slow and prone to fail for a host of unrelated reasons)

if there's something I'm missing I'm open to hearing it, but I think trying to force a test for this is a net loss for us.